multicore_runtime/wsgi.py

```diff
@@ -43,6 +43,19 @@
 
 root_logger.setLevel(logging.INFO)
 
+# Enable Python Cloud Debugger if one is present. The debugger agent
+# communicates with the Cloud Debugger backend.
+#
+# The debugger has no effect on the application until snapshot is
+# requested.
+#
+# For details see: https://github.com/GoogleCloudPlatform/cloud-debug-python
+try:
```

Comment:
Correct. You can use the debugger in local dev mode. The reason we added this check in Java was because Java Cloud Debugger can't work together with regular debugger (JDWP).